nixos/modules/services/networking/gns3-server.nix

```diff
@@ -226,37 +232,6 @@ in {
           ++ lib.optional cfg.ubridge.enable "ubridge";
         User = "gns3";
         WorkingDirectory = "%S/gns3";
-
-        # Hardening
```

Comment:
Add a comment stating that these SystemD hardenings are incompatible with GNS3 (DynamicUsers, and NoNewPrivileges)